README.md

```diff
@@ -89,7 +108,12 @@ entities (no need to draw an ERD):
 
 Your answer:
 ```
-Replace this with your answer
+An ERD (Entity Relationship Diagram) is a great tool to use when think about different relationship between data in a data base. Relationships can
+exist in many forms including: one-to-many, one-to-one, or many-to-one.
+
+The example above(Genie, Lamp; Person, Pet) is a perfect example of  relationships: A genie may come from one lamp; a lamp may belong to many
+people; a person may have on or many pets -- usually the pet only has one owner. The owner could only have one pet, which would make that a
+one-to-one relationship since pets usually only have one owner.
 ```
 
 ### Question 5
```

Comment:
For quizzes I strongly encourage you to leave an answer/comments, even if it's wrong. That way I get a better idea of where you're at with the material.

Is there anything in particular about this question that I can help out with?